ui/core/proto_utils/logs_parser.tsx

```diff
@@ -771,6 +776,14 @@ export class ResourceChangedLogGroup extends SimLog {
 	static fromLogs(logs: Array<SimLog>): Record<ResourceType, Array<ResourceChangedLogGroup>> {
 		const allResourceChangedLogs = logs.filter((log): log is ResourceChangedLog => log.isResourceChanged());
 
+		const maxRessource = function(logs: ResourceChangedLog[]) {
+			let max : number | null = null
```

Comment:
Could potentially allow for this to just be a `number` by adding totals to the other resource types too. Might clean it up a bit but I don't feel that strongly about it